examples/goldilocks_models/gtest_initial_guess.cc

```diff
@@ -0,0 +1,90 @@
+#include <gtest/gtest.h>
```

Comment:
Rename `gtest_initial_guess.cc` to `initial_guess_test.cc` to follow the naming convention